docs/_templates/template_ft_in_action.md

```diff
@@ -0,0 +1,104 @@
+```
+
+
+## Choosing the model
+Always show the available models in your guide. Then mention which model will be used in your fine-tuning task. 
+Feel free to add a `See Also` {admonition} for supplementary info on the model, perhaps a relevant paper or site.
+
+Example:
+
+"You can see all available models either in [the docs](../2_step_by_step/2_5_choose_back_bone.md) or by calling:"
+```python
+finetuner.describe_models()
+```
+
+```bash
+                                                                  Finetuner backbones                                                                   
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃                                            model ┃           task ┃ output_dim ┃ architecture ┃                                          description ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│                                         resnet50 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                        resnet152 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b0 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b4 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                     openai/clip-vit-base-patch32 │  text-to-image │        768 │  transformer │ Pretrained on millions of text image pairs by OpenAI │
+│                                  bert-base-cased │   text-to-text │        768 │  transformer │       Pretrained on BookCorpus and English Wikipedia │
+│ sentence-transformers/msmarco-distilbert-base-v3 │   text-to-text │        768 │  transformer │           Pretrained on Bert, fine-tuned on MS Marco │
+└──────────────────────────────────────────────────┴────────────────┴────────────┴──────────────┴──────────────────────────────────────────────────────┘
+```
+
+
+## Creating a fine-tuning job
+Show the user how to create a fine-tuning run, then explain why your example run has particular parameters and what they do. Also mention which parameters are optional or required.
+Provide a more detailed explanation of parameters that are important for your particular experiment. 
+
+Example:
+
+```python
+run = finetuner.fit(
+    ...
+    )
+```
+"Let's understand what this piece of code does ..."
+
+
+Also show the user how they can monitor their run. Example:
+
+"Now that we've created a run, let's see its status."
+```python
+print(run.status())
+```
+
+```bash
+{'status': 'CREATED', 'details': 'Run submitted and awaits execution'}
+```
```

Comment:
And if we'll have "Monitor the run" section this can go there + showing that reconnecting is possible and easy.